addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -231,6 +231,7 @@ func get_interact_position() -> Vector2:
 	for c in get_children():
 		if c is Position2D:
 			if c.get_owner() == self:
+				interact_position = global_position
```

Comment:
Idk. I guess it got removed in one of the optimization steps. I think I remember something about something not being used somewhere or something. I really don't know.

Reply:
Ok, let's keep an eye on this. I don't believe this will bring some hideous bug or anything, but I just don't remember why we just `continue`d if L233 was true, without doing anything else.